index.html

```diff
@@ -1,203 +1,233 @@
+          </div>
+
+          <div class="language-list">
+            <div class="same">
+              <img src="./img/js.png" alt="">
+              <span class="lists">Javascript</span>
+            </div>
+            <div class="same">
+              <img src="./img//html.png" alt="">
+              <span class="lists">HTML</span>
+            </div>
+            <div class="same">
+              <img src="./img//css.png" alt="">
+              <span class="lists">CSS</span>
+            </div>
+          </div>
+
+          <div class="language">
+            <span class="languages">Frameworks</span> <span class="material-icons">chevron_right</span>
+          </div>
+          <div class="language">
+            <span class="languages">Skills</span> <span class="material-icons">chevron_right</span>
+          </div>
+
         </div>
-      </div>
-
-      <div class="language">
-        <span class="languages">Frameworks</span> <span class="material-icons">chevron_right</span>
-      </div>
-      <div class="language">
-        <span class="languages">Skills</span> <span class="material-icons">chevron_right</span>
-      </div>
-      
-    </div>
-
-  </section>
-
-  <section class="contacts">
-    <h2 class="contact-me">Contact Me</h2> 
-    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
-    <form action="https://formspree.io/f/meqvlald" method="POST">
-      <label >
-        <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>
-      </label>
-      <label class="mail-input">
-        <input type="email" id="email" placeholder="Email" name="_replyto" required>
-      </label>
-      <label>
-      <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write Your message here" required></textarea>
-      </label>
-      <button type="submit">Get in Touch</button>
-    </form>
-  </section>
-</body>
+
+      </section>
+
+      <section class="contacts">
+        <h2 class="contact-me">Contact Me</h2>
+        <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a
+          <br>
+          project that needs coding. I’d <br> love to help with it </p>
+        <form action="https://formspree.io/f/meqvlald" method="POST">
+          <label>
+            <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>
+          </label>
+          <label class="mail-input">
+            <input type="email" id="email" placeholder="Email" name="_replyto" required>
+          </label>
+          <label>
+            <textarea name="message" id="message" cols="30" rows="10" maxlength="500"
+              placeholder="Write Your message here" required></textarea>
+          </label>
+          <button type="submit">Get in Touch</button>
+        </form>
+      </section>
```

Comment:
- [ ] You should stick the design. Your design background image is different. Please fix this.
- [ ] End of the page is empty. I can scroll down.You should fix this.

**Original design:**
![image](https://user-images.githubusercontent.com/38635158/125698480-52978b59-fac9-4c9d-b270-b7b770c9014e.png)
___
**Your design:**
![image](https://user-images.githubusercontent.com/38635158/125699002-d64bdc6a-dd5c-4e4f-9eaa-072465ca615d.png)